### PR TITLE
Fix/available watchers

### PR DESCRIPTION
**Pull request by ulferts**
https://community.openproject.org/work_packages/17645

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://fix-available-watchers.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: spec/requests/api/v3/watcher_resource_spec.rb [hunk 1]
```diff
@@ -152,6 +158,46 @@
         end
       end
     end
+  end
+
+  describe '#available_watchers' do
+    subject(:response) { last_response }
+    let(:authorized_user) do
+      FactoryGirl.create :user,  member_in_project: project,
+                                 member_through_role: add_watchers_role
```
Comment (hschink) [inline spec/requests/api/v3/watcher_resource_spec.rb]: _Wuff wuff_, _bark bark_ (you see I'm Hound): I just want to mention the additional space between `:user,` and the following key.

Comment (ulferts) [inline spec/requests/api/v3/watcher_resource_spec.rb]: Thanks

File: spec/requests/api/v3/watcher_resource_spec.rb [hunk 2]
```diff
@@ -152,6 +158,46 @@
         end
       end
     end
+  end
+
+  describe '#available_watchers' do
```
Comment (hschink) [inline spec/requests/api/v3/watcher_resource_spec.rb:163]: @ulferts Please excuse that I just recognize now that you might want to use [the shared example `API V3 collection decorated`](https://github.com/opf/openproject/blob/dev/spec/lib/api/v3/support/api_v3_collection.rb). See [PriorityRepresenter's spec](https://github.com/opf/openproject/blob/dev/spec/lib/api/v3/priorities/priority_collection_representer_spec.rb) for an example.

Comment (hschink) [inline spec/requests/api/v3/watcher_resource_spec.rb:163]: You're right it's a different kind of test here. No need to fix anything here.

**Pull request merged**